dftimewolf/lib/collectors/grr_hunt.py

```diff
@@ -153,31 +242,34 @@ class GRRHuntFileCollector(GRRHunt):
 
   def __init__(self,
                state: DFTimewolfState,
-               name: Optional[str]=None,
-               critical: bool=False) -> None:
+               name: Optional[str] = None,
+               critical: bool = False) -> None:
     """Initializes a GRR file collector hunt.
 
     Args:
       state (DFTimewolfState): recipe state.
       name (Optional[str]): The module's runtime name.
-      critical (Optional[bool]): True if the module is critical, which causes
+      critical (bool): True if the module is critical, which causes
           the entire recipe to fail if the module encounters an error.
     """
     super(GRRHuntFileCollector, self).__init__(
         state, name=name, critical=critical)
     self.file_path_list = []  # type: List[str]
     self.max_file_size = 5*1024*1024*1024  # 5 GB
 
-  # pylint: disable=arguments-differ
+  # pylint: disable=arguments-differ,too-many-arguments
   def SetUp(self,
             file_path_list: str,
             reason: str,
             grr_server_url: str,
             grr_username: str,
             grr_password: str,
             max_file_size: str,
-            approvers: Optional[str]=None,
-            verify: bool=True) -> None:
+            approvers: Optional[str] = None,
+            verify: bool = True,
+            match_mode: Optional[str] = None,
+            client_operating_systems: Optional[str] = None,
+            client_labels: Optional[str] = None) -> None:
```

Comment:
Please see my note in the other SetUp function

Reply:
Cleaned up.